release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -858,6 +858,14 @@ clouds:
 
 To use application credentials with your cluster as a {rh-openstack} administrator, create the credentials. Then, use them in a `clouds.yaml` file when you install a cluster. Alternatively, you can create the `clouds.yaml` file and rotate it into an existing cluster.
 
+[id="ocp-4-12-hcp"]
+=== Hosted control planes (Technology Preview)
+
+[id="ocp-4-12-hcp-versioning"]
+==== Versioning support for hosted control planes
+
+With each major, minor, or patch version release of {product-title}, 3 components of hosted control planes are released: the HyperShift Operator, the `HostedCluster` and `NodePool` API resources, and the command-line interface (CLI). Any HyperShift Operator or API resources that are released for a minor _n_ version of {product-title} must support _n_, _n-1_, and _n-2_ versions of {product-title}. For more information, see xref:../architecture/control-plane.html#hosted-control-planes-version-support_control-plane[Versioning support for hosted control planes].
```

Comment:
should we qualify this stmt?

do we need to qualify this stmt so that the min release version supported begins at GA of the hosted control plane technology and is not extended to prior releases?

Reply:
@derekwaynecarr Good question. Adel and I discussed adding an example to show real version numbers instead of simply saying _n_, _n-1_, and _n-2_, but hosted control planes haven't been out long enough to do that. I see your point of not wanting to imply that the support extends beyond the GA of hosted control planes. I can add some wording to clarify that. Maybe something like this? "Currently, because hosted control planes has been available only since version 4.11, support extends as far back as version 4.11."